biosys/apps/main/api/uploaders.py

```diff
@@ -3,16 +3,18 @@
 import datetime
 from os import path
 
+import datapackage
 from django.conf import settings
 from django.utils import six, timezone
 from openpyxl import load_workbook
+from slugify import slugify
```

Comment:
No need to change but I think django has a slugify utils function these days so you don't need the slugify package

Reply:
correct `django.utils.text.slugify()`. I will change that in my next update.